### PR TITLE
SRCH-799 Create mock Single Search Endpoint for CKS

### PR DESCRIPTION
**Pull request by dcmwong**
https://nicedigital.atlassian.net/browse/SRCH-799

### CODE REVIEW

**Comment by dcmwong** [comment]
Open to changing the default port.  I've put it to 3000 as most Node projects I've worked on default to that port.

File: search-apis/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
+FROM node:14-alpine
```
Comment (ediblecode) [inline search-apis/Dockerfile]: Can we go straight to Node 18? Just cos 14 is EOL in 4 months...

File: search-apis/package.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@nice-digital/cks-fake-elastic",
```
Comment (ediblecode) [inline search-apis/package.json]: Can we call this something like `@nice-digital/mock-search-api` please?

File: search-apis/package.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    "test:watch": "jest --watchAll",
+    "dev": "nodemon index.js"
+  },
+  "author": "NICE Digital (https://www.nice.org.uk/digital)",
+  "contributors": [
+    "NICE Digital Services (https://www.nice.org.uk/digital)",
+    "David Wong"
+  ],
+  "license": "MIT",
+  "homepage": "https://github.com/nice-digital/cks-gatsby/functional-tests/fake-elastic/#readme",
```
Comment (ediblecode) [inline search-apis/package.json]: Can you update the homepage URL to the new mock APIs repo sub folder?

File: search-apis/package.json [hunk 4]
```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@nice-digital/cks-fake-elastic",
+  "version": "1.0.0",
+  "description": "Fake ElasticSearch instance for CKS automated tests",
```
Comment (ediblecode) [inline search-apis/package.json]: Can you update the description?

**Review by ediblecode — CHANGES_REQUESTED** [review]

**Review by colinmazu — APPROVED** [review]

**Review by ediblecode — APPROVED** [review]

**Pull request merged**